Documentation/ApiOverview/Enumerations/Index.rst

```diff
@@ -19,3 +23,23 @@ methods of a BitSet can.
 
    Enumeration
    BitSet
+
+
+Background and history
+======================
+
+PHP has no enumeration concept as part of the language up to date. Therefore
+the TYPO3 CMS core includes a custom enumeration implementation.
+
+An enumeration or bitset should be preferred over using pure constants, as the
```

Comment:
This duplicates line 17, I would remove it here and leave it above, as the background/history part is purely additional info, the constants hint however one of the most important things with our usage of enums.